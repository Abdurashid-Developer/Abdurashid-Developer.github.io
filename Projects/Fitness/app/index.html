<!DOCTYPE html>
<html lang="ru">

<head>
    <!-- Необходимые Мета-теги всегда на первом месте -->
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Fitness</title>

    <!-- Bootstrap & Fontawesome CDN CSS -->
    <link rel="stylesheet" href="libs/css/bootstrap.min.css">
    <link href="libs/css/font-awesome.css" rel="stylesheet">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="libs/wow/animate.css">
    <link rel="stylesheet" href="libs/css/pe-icon-7-stroke.css">

</head>


<body>
  <!--header -->
    <section class="header" id="Home">
        <div class="menu-top">
            <nav class="navbar navbar-expand-lg navbar-light ">
                <div class="logo">
                    <div class="row justify-content-lg-start">
                        <img src="./img/logo icon.png" alt="">
                        <h3>fitness</h3>
                    </div>
                </div>
                <div class="container">
                    <a class="navbar-brand" href="#"></a>
                    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav"
                        aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
                        <span class="navbar-toggler-icon"></span>
                    </button>
                    <div class="collapse navbar-collapse" id="navbarNav">
                        <ul class="navbar-nav ml-auto">
                            <li class="nav-item active">
                                <a class="nav-link" href="#Home">Home<span class="sr-only">(current)</span></a>
                            </li>
                            <li class="nav-item active">
                                <a class="nav-link" href="#About">About<span class="sr-only">(current)</span></a>
                            </li>
                            <li class="nav-item">
                                <a class="nav-link" href="#Works">How it Works</a>
                            </li>
                            <li class="nav-item">
                                <a class="nav-link" href="#Services">Services</a>
                            </li>
                            <li class="nav-item">
                                <a class="nav-link" href="#FAQ">FAQ</a>
                            </li>
                            <li class="nav-item">
                                <a class="nav-link" href="#Contact">Contact</a>
                            </li>
                        </ul>
                    </div>
                </div>
            </nav>
        </div>
    </section>
 <!--end header -->
 
     <!--can -->
    <section class="can">
        <div class="container">
            <div class="row ">
                <div class="text-menu ">
                    <div class="row ">
                        <div class="gr">
                            <h1>Fitness in a can <span> / </span></h1>
                        </div>
                    </div>
                    <div class="row ">
                        <div class="red">
                            <p>Yes you can <span>/</span> </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
   <!--end can -->
   
     <!--green -->
    <section class="green">
        <div class="container">
            <div class="col-lg-12">
                <div class="info">
                    <div class="row justify-content-center" id="About">
                        <div class="col-lg-3 col-md-6 wow bounceInLeft" data-wow-delay="100ms">
                            <i class="fa fa-suitcase" aria-hidden="true"></i>
                            <h3>Resources</h3>
                            <p>Easily manage your inspiration and <br> work-in-progress by dragging images into <br>
                                projects and
                                sharable client groups. Easily <br> manage your inspiration and work-in-progress</p>
                            <p>Explore more →</p>
                        </div>
                        <div class="col-lg-3 col-md-6 wow bounceInLeft" data-wow-delay="500ms">
                            <i class="fa fa-clock-o" aria-hidden="true"></i>
                            <h3>Training & Funding</h3>
                            <p>Easily manage your inspiration and <br> work-in-progress by dragging images into <br>
                                projects and
                                sharable client groups. Easily <br> manage your inspiration and work-in-progress</p>
                            <p>Browse training →</p>
                        </div>
                        <div class="col-lg-3 col-md-6 wow bounceInRight" data-wow-delay="900ms">
                            <i class="fa fa-mixcloud" aria-hidden="true"></i>
                            <h3>Connect</h3>
                            <p>Easily manage your inspiration and <br> work-in-progress by dragging images into <br>
                                projects and
                                sharable client groups. Easily <br> manage your inspiration and work-in-progress</p>
                            <p>Connect now →</p>
                        </div>
                        <div class="col-lg-3 col-md-6 wow bounceInRight" data-wow-delay="1200ms">
                            <i class="fa fa-comment" aria-hidden="true"></i>
                            <h3>Communicate</h3>
                            <p>Easily manage your inspiration and <br> work-in-progress by dragging images into <br>
                                projects and
                                sharable client groups. Easily <br> manage your inspiration and work-in-progress</p>
                            <p>Start now →</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <!--end green -->
    
   <!--//girl// -->
    <section class="girl" id="Works">
        <div class="container">
            <div class="col-lg-12">
                <div class="row justify-content-center no-gutters">
                    <div class="col-lg-4 wow rotateInDownLeft" data-wow-delay="100ms">
                        <div class="first">
                            <div class="it">
                                <p><span>it’s all about</span></p>
                            </div>
                            <h2>fitness first</h2>
                            <div class="row justify-content-center">
                                <i class="fa fa-dropbox" aria-hidden="true"></i>
                            </div>
                            <p>Lorem ipsum dolor sit amet, consectetur <br> adipisicing elit, sed do eiusmod tempor
                                <br>
                                incididunt
                                ut labore et dolore magna aliqua. <br>
                                Ut enim ad minim veniam, quis nostrud. <br> <br>
                                <span>Read More</span> </p>
                        </div>
                    </div>
                    <div class="col-lg-8 no-gutters wow rotateInDownRight" data-wow-delay="500ms ">
                        <img src="./img/girl.png" alt="">

                        <div class="row justify-content-end">
                            <div class="Ac ">
                                <p> <span> Active Users</span> (34)</p>
                            </div>
                        </div>
                    </div>
                </div>
                <div class="row justify-content-center">
                    <div class="col-lg-8 no-gutters wow rotateInDownLeft" data-wow-delay="900ms">
                        <img src="./img/hand.png" alt="">
                        <div class="Ac ">
                            <p> <span>Products USed</span>(658)</p>
                        </div>
                    </div>
                    <div class="col-lg-4 no-gutters wow rotateInDownRight" data-wow-delay="1200ms">
                        <div class="first-1">
                            <div class="love">
                                <p><span>love yuor</span></p>
                            </div>
                            <h2>Your body</h2>
                            <div class="row justify-content-center">
                                <i class="fa fa-dropbox" aria-hidden="true"></i>
                            </div>
                            <p>Lorem ipsum dolor sit amet, consectetur <br> adipisicing elit, sed do eiusmod tempor
                                <br>
                                incididunt
                                ut labore et dolore magna aliqua. <br>
                                Ut enim ad minim veniam, quis nostrud. <br> <br>
                                <span>Read More</span> </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
  <!--//end girl// -->
  
  <!--//beach// -->
    <section class="beach" id="Services">
        <div class="container">
            <div class="col-lg-12">
                <div class="row justify-content-center ">
                    <div class="col-lg-4 wow rotateIn" data-wow-delay="100ms">
                        <div class="cross">
                            <img src="./img/cross.png" alt="">
                        </div>
                    </div>
                    <div class="col-lg-8  wow rotateIn" data-wow-delay="500ms ">
                        <div class="man">
                            <img src="./img/beach.png" alt="">
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
   <!--//end beach// -->
   
     <!--//summer// -->
    <section class="summer" id="">
        <div class="container">
            <div class="col-lg-12">
                <div class="row text-center">
                    <div class="col-lg-4 no-gutters wow rollIn" data-wow-delay="100ms">
                        <div class="track">
                            <p><span>track your</span> </p>
                            <div class="row justify-content-center">
                                <img src="./img/man.png" alt="">
                            </div>
                            <h2>Life Style</h2>
                            <p>Lorem ipsum dolor sit amet, consectetur <br> adipisicing elit, sed do eiusmod tempor
                                <br>
                                incididunt
                                ut labore et dolore magna aliqua. <br>
                                Ut enim ad minim veniam, quis nostrud. <br>
                            </p>
                        </div>
                    </div>
                    <div class="col-lg-8 no-gutters  wow rollIn" data-wow-delay="500ms">
                        <div class="sum">
                            <br><br>
                            <h2>Summer of fun</h2>
                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
                                incididunt ut labore et <br> dolore magna aliqua. Ut enim ad minim veniam, quis
                                nostrud.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
  <!-- //end summer//   -->
  
  <!--//fruct// -->
    <div class="fruct">
        <div class="container">
            <div class="col-lg-12 no-gutters">
                <div class="row justify-content-center">
                    <div class="col-lg-8 col-md-8 wow rotateInDownLeft" data-wow-delay="100ms">
                        <div class="grapes">
                            <img src="./img/grapes.png" alt="">
                        </div>
                    </div>
                    <div class="col-lg-4 col-md-4 wow rotateInDownRight" data-wow-delay="100ms">
                        <div class="mango">
                            <img src="./img/mango.png" alt="">
                        </div>
                    </div>
                </div>
                <div class="row justify-content-center">
                    <div class="col-lg-8 wow rotateInUpLeft" data-wow-delay="100ms">
                        <div class="get">
                            <p><span>track your</span> </p>
                            <h2> get involved in yourself </h2>
                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
                                incididunt ut <br> labore et dolore magna aliqua. Ut enim ad minim veniam, quis
                                nostrud. ex
                                ea commodo consequat. <br> Duis aute irure dolor in reprehenderit in voluptate velit
                                esse
                                cillumdolore eu fugiat nulla pariatur
                            </p>
                        </div>
                    </div>
                    <div class="col-lg-4 wow rotateInUpRight" data-wow-delay="100ms">
                        <div class="habits">
                            <p>improve your</p>
                            <h2>eating habits </h2>
                            <p>Lorem ipsum dolor sit amet, consectetur <br> adipisicing elit, sed do eiusmod tempor <br>
                                incididunt
                                ut labore et dolore magna aliqua. <br>
                                Ut enim ad minim veniam, quis nostrud.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
  <!--//end fruct// --> 
    
    <!--//night// -->
    <section class="night" id="FAQ">
        <div class="container-fluid">
            <div class="row text-center">
                <div class="col-lg-12 wow bounceInLeft" data-wow-delay="300ms">
                    <h1>We launch leaders with big ideas</h1>
                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do <br> eiusmod tempor incididunt
                        ut
                        labore et dolore magna aliqua.</p>
                </div>
            </div>
        </div>
    </section>
    <!--//end night// -->
    
   <!--  //footer// -->
    <section class="footer" id="Contact">
        <div class="container">
            <div class="row justify-content-center">
                <div class="col-lg-12">
                    <div class="row ">
                        <div class="col-lg-4 col-md-4 wow fadeInUp" data-wow-delay="100ms">
                            <p> Resourcesp</p>
                            <p>Easily manage your inspiration and work-in-progress by <br> dragging images into
                                projects and
                                sharable client groups. <br> work-in-progress by dragging images into projects and <br>
                                sharable
                                client group</p>
                        </div>
                        <div class="col-lg-4 col-md-4 wow fadeInUp" data-wow-delay="300ms">
                            <p> About us</p>
                            <p>Easily manage your inspiration and work-in-progress by dragging <br> images into
                                projects and
                                sharable client groups. work-in-progress by dragging images into projects and
                                sharable
                                client group</p>
                        </div>
                        <div class="col-lg-4 col-md-4 wow bounceInRight" data-wow-delay="600ms">
                            <div class="row  justify-content-center">
                                <p>Stay in touch</p>
                            </div>

                            <div class="icon">
                                <div class="row justify-content-center">
                                    <i class="fa fa-facebook" aria-hidden="true"></i>
                                    <i class="fa fa-twitter" aria-hidden="true"></i>
                                    <i class="fa fa-youtube-play" aria-hidden="true"></i>
                                    <i class="fa fa-inr" aria-hidden="true"></i>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div class="row justify-content-center">
                        <div class="col-lg-12">
                            <div class="form">
                                <form action="">
                                    <div class="row justify-content-center ">
                                        <div class="col-lg-4 col-md-4"></div>
                                        <div class="col-lg-2 col-md-2 wow bounceInLeft" data-wow-delay="1200ms">
                                            <input type="text" class="form-control" placeholder="Your Name">
                                        </div>
                                        <div class="col-lg-4 col-md-4 wow bounceInLeft" data-wow-delay="800ms">
                                            <input type="text" class="form-control" placeholder="Your email">
                                        </div>
                                        <div class="col-lg-2 col-md-2 wow bounceInRight" data-wow-delay="1500ms">
                                            <div class="join">
                                                <input type="text" class="form-control" placeholder="Join Newsletter">
                                            </div>
                                        </div>
                                    </div>
                                </form>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
   <!--//end footer// -->

    <!--     <a id="backtotop" href="#top">
        <i class="fa fa-chevron-up"></i>
    </a> -->
    
    
    <!--scroll -->
    <a id="scrollUp" href="#top" style="position: fixed; z-index: 2147483647;"></a>

    <!--scripts -->

    <script src="libs/js/jquery.min.js"></script>
    <script src="layout/scripts/jquery.backtotop.js"></script>
    <script src="layout/scripts/jquery.mobilemenu.js"></script>
    <script src="libs/js/popper.js"></script>
    <script src="libs/js/bootstrap.js"></script>
    <script src="js/script.js"></script>
    <script src="libs/wow/wow.min.js"></script>
    <script src="libs/js/jquery.scrollUp.min.js"></script>

</body>

</html>