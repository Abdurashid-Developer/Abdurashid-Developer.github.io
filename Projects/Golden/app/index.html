<!DOCTYPE html>
<html lang="ru">

<head>
    <!-- Необходимые Мета-теги всегда на первом месте -->
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Golden</title>

    <!-- Bootstrap & Fontawesome CDN CSS -->
    <link rel="stylesheet" href="libs/css/bootstrap.min.css">
    <link href="libs/css/font-awesome.css" rel="stylesheet">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="libs/css/pe-icon-7-stroke.css">
    <link rel="stylesheet" href="libs/wow/animate.css">
</head>

<body>
    <!--header-->
    <section class="header" id="Home">
        <div class="menu-top">
            <nav class="navbar navbar-expand-lg navbar-light ">
                <div class="logo">
                    <div class="row ">
                        <img src="./img/Logo-Treehouse.png" alt="">
                    </div>
                </div>
                <!--container -->
                <div class="container">
                    <a class="navbar-brand" href="#"></a>
                    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav"
                        aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
                        <span class="navbar-toggler-icon"></span>
                    </button>
                    <div class="collapse navbar-collapse" id="navbarNav">
                        <ul class="navbar-nav ml-auto">
                            <li class="nav-item">
                                <a class="nav-link" href="#Home"> Home </a>
                            </li>
                            <li class="nav-item">
                                <a class="nav-link" href="#Services"> Services </a>
                            </li>
                            <li class="nav-item">
                                <a class="nav-link" href="#portfolio">portfolio</a>
                            </li>
                            <li class="nav-item">
                                <a class="nav-link" href="#about">about</a>
                            </li>

                            <li class="nav-item">
                                <a class="nav-link" href="#contact">contact</a>
                            </li>
                        </ul>
                    </div>
                </div> <!--  end container -->
            </nav>
        </div>
        <div class="container">
            <div class="col-lg-12">
                <div class="row  justify-content-center ">
                    <div class="text-menu text-center">
                        <h3>Welcome To Our Studio!</h3>
                        <h1>It’s nice to Meet you</h1>
                    </div>
                </div>
                <div class="row justify-content-center">
                    <button class="btn">
                        <p>Tell me more </p>
                    </button>
                </div>
            </div>
        </div>
    </section>
    <!-- end header-->

    <!--services-->
    <section class="services" id="Services">
        <div class="container">
            <div class="row text-center">
                <div class="col-lg-12">
                    <h2>Services</h2>
                    <p>Proin iaculis purus consequat sem cure.</p>
                    <div class="row justify-content-center">
                        <div class="col-lg-4 col-md-4 wow slideInLeft" data-wow-delay="100ms">
                            <i class="fa fa-shopping-basket" aria-hidden="true"></i>
                            <h4>E-Commerce</h4>
                            <p>Proin iaculis purus consequat sem cure <br>
                                digni ssim. Donec porttitora entum suscipit <br>
                                aenean rhoncus posuere odio in tincidunt.</p>
                        </div>
                        <div class="col-lg-4 col-md-4 wow fadeInUp" data-wow-delay="500ms">
                            <i class="fa fa-laptop" aria-hidden="true"></i>
                            <h4>Responsive Web</h4>

                            <p>Proin iaculis purus consequat sem cure <br>
                                digni ssim. Donec porttitora entum suscipit <br>
                                aenean rhoncus posuere odio in tincidunt.</p>
                        </div>
                        <div class="col-lg-4 col-md-4  wow slideInRight" data-wow-delay="900ms">
                            <i class="fa fa-lock" aria-hidden="true"></i>
                            <h4>Web Security</h4>
                            <p>Proin iaculis purus consequat sem cure <br>
                                digni ssim. Donec porttitora entum suscipit <br>
                                aenean rhoncus posuere odio in tincidunt.</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <!--end services-->

    <!--portfolio-->
    <section class="portfolio" id="portfolio">
        <div class="container">
            <div class="row text-center">
                <div class="col-lg-12">
                    <h2>Our portfolio</h2>
                    <p>Proin iaculis purus consequat sem cure.</p>
                    <div class="row justify-content-center">
                        <div class="col-lg-4 col-md-4 sss wow fadeInUp" data-wow-delay="100ms ">
                            <div class="our  justify-content-center">
                                <img src="./img/portfolio1.png" alt="">
                                <h4>Ebony & Ivory</h4>
                                <p>Branding</p>
                            </div>
                        </div>
                        <div class="col-lg-4 col-md-4  sss wow fadeInUp" data-wow-delay="300ms">
                            <div class="our">
                                <img src="./img/portfolio2.png" alt="">
                                <h4>Smart Stationary</h4>
                                <p>Print Design</p>
                            </div>
                        </div>

                        <div class="col-lg-4 col-md-4 sss wow fadeInUp" data-wow-delay="600ms ">
                            <div class="our">
                                <img src="./img/portfolio3.png" alt="">
                                <h4>Clever Poster</h4>
                                <p>Print Design</p>
                            </div>
                        </div>
                    </div>
                    <div class="row text-center">
                        <div class="col-lg-4 col-md-4  sss wow fadeInUp" data-wow-delay="900ms">
                            <div class="our ">
                                <img src="./img/portfolio4.png" alt="">
                                <h4>Ebony & Ivory</h4>
                                <p>Branding</p>
                            </div>
                        </div>
                        <div class="col-lg-4 col-md-4 sss wow fadeInUp" data-wow-delay="1100ms">
                            <div class="our">
                                <img src="./img/portfolio5.png" alt="">
                                <h4>Smart Stationary</h4>
                                <p>Print Design</p>
                            </div>
                        </div>
                        <div class="col-lg-4 col-md-4 sss wow fadeInUp" data-wow-delay="1400ms">
                            <div class="ourr">
                                <img src="./img/portfolio6.png" alt="">
                                <h4>Clever Poster</h4>
                                <p>Print Design</p>
                            </div>
                        </div>
                    </div>
                </div>

            </div>
        </div>
    </section>
    <!--end portfolio-->

    <!--about-->
    <section class="about" id="about">
        <div class="container">
            <div class="row text-center">
                <div class="col-lg-12">
                    <h2>about us</h2>
                    <p>Proin iaculis purus consequat sem cure. </p>
                    <div class="row justify-content-center">
                        <div class="col-lg-4 col-md-4 wow fadeInLeft" data-wow-delay="100ms">
                            <h4>July 2010 <br>
                                Our Humble Beginnings</h4>
                            <p>Proin iaculis purus consequat sem cure
                                digni ssim. Donec porttitora entum suscipit
                                aenean rhoncus posuere odio in tincidunt. Proin
                                iaculis purus consequat sem cure digni
                                ssim. Donec porttitora entum suscipit.</p>
                        </div>
                        <div class="col-lg-4 col-md-4 wow fadeInUp" data-wow-delay="300ms">
                            <img src="./img/about1.png" alt="">
                        </div>
                        <div class="col-lg-4 col-md-4"></div>
                    </div>
                    <div class="row justify-content-center">
                        <div class="col-lg-4 col-md-4"></div>
                        <div class="col-lg-4 col-md-4  wow fadeInUp" data-wow-delay="600ms">
                            <img src="./img/about2.png" alt="">
                        </div>
                        <div class="col-lg-4  wow fadeInRight" data-wow-delay="900ms">
                            <h4>July 2010 <br>
                                Our Humble Beginnings</h4>
                            <p>Proin iaculis purus consequat sem cure
                                digni ssim. Donec porttitora entum suscipit
                                aenean rhoncus posuere odio in tincidunt. Proin
                                iaculis purus consequat sem cure digni
                                ssim. Donec porttitora entum suscipit.</p>
                        </div>
                    </div>
                    <div class="row justify-content-center">
                        <div class="col-lg-4 col-md-4  wow fadeInUp" data-wow-delay="1200ms">
                            <h4>July 2016 <br>
                                Our Humble Beginnings</h4>
                            <p>Proin iaculis purus consequat sem cure
                                digni ssim. Donec porttitora entum suscipit
                                aenean rhoncus posuere odio in tincidunt. Proin
                                iaculis purus consequat sem cure digni
                                ssim. Donec porttitora entum suscipit.</p>
                        </div>
                        <div class="col-lg-4 col-md-4 wow fadeInLeft" data-wow-delay="1400ms">
                            <img src="./img/about3png.png" alt="">
                        </div>
                        <div class="col-lg-4"></div>
                    </div>
                    <div class="row justify-content-center">
                        <div class="col-lg-4 col-md-4"></div>
                        <div class="col-lg-4 col-md-4 wow fadeInUp" data-wow-delay="1600ms">
                            <img src="./img/about4.png" alt="">
                        </div>
                        <div class="col-lg-4 col-md-4 wow fadeInRight" data-wow-delay="1800ms">
                            <h4>July 2018 <br>
                                Our Humble Beginnings</h4>
                            <p>Proin iaculis purus consequat sem cure
                                digni ssim. Donec porttitora entum suscipit
                                aenean rhoncus posuere odio in tincidunt. Proin
                                iaculis purus consequat sem cure digni
                                ssim. Donec porttitora entum suscipit.</p>
                        </div>
                    </div>
                    <div class="row justify-content-center">
                        <div class="col-lg-4 col-md-4"></div>
                        <div class="col-lg-4 col-md-4 wow fadeInUp" data-wow-delay="2000ms">
                            <img src="./img/Our Story Continues.png" alt="">
                        </div>
                        <div class="col-lg-4 col-md-4"></div>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <!--end about-->

    <!--team-->
    <section class="team">
        <div class="container">
            <div class="row text-center">
                <div class="col-lg-12">
                    <h2>our amazing team</h2>
                    <p>Proin iaculis purus consequat sem cure. </p>
                    <div class="row justify-content-center">
                        <div class="col-lg-4 col-md-4 wow fadeInUp" data-wow-delay="100ms">
                            <img src="./img/our.png" alt="">
                            <h4>Kimberly Thompson</h4>
                            <p>Marketer</p>
                            <i class="fa fa-twitter" aria-hidden="true"></i>
                            <i class="fa fa-facebook" aria-hidden="true"></i>
                            <i class="fa fa-google" aria-hidden="true"></i>
                        </div>
                        <div class="col-lg-4 col-md-4 wow fadeInUp" data-wow-delay="300ms">
                            <img src="./img/our.png" alt="">
                            <h4>Rico Massimo</h4>
                            <p>Coder</p>
                            <i class="fa fa-twitter" aria-hidden="true"></i>
                            <i class="fa fa-facebook" aria-hidden="true"></i>
                            <i class="fa fa-google-plus" aria-hidden="true"></i>
                        </div>
                        <div class="col-lg-4 col-md-4 wow fadeInUp" data-wow-delay="600ms">
                            <img src="./img/our.png" alt="">
                            <h4>Uku Mason </h4>
                            <p>Graphic Designer </p>
                            <i class="fa fa-twitter" aria-hidden="true"></i>
                            <i class="fa fa-facebook" aria-hidden="true"></i>
                            <i class="fa fa-telegram" aria-hidden="true"></i>
                        </div>
                    </div>
                    <div class="our-end">
                        <p>Proin iaculis purus consequat sem cure digni ssim donec porttitora entum suscipit <br>
                            aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <!-- endteam-->

    <!--info -->
    <section class="info">
        <div class="container">
            <div class="col-lg-12">
                <div class="row justify-content-center">
                    <div class="col-lg-3 col-md-6 text-center wow fadeInUp" data-wow-delay="100ms">
                        <img src="./img/Envato.png" alt="">
                    </div>
                    <div class="col-lg-3 col-md-6 text-center wow fadeInUp" data-wow-delay="300ms">
                        <img src="./img/WordPress.png" alt="">
                    </div>

                    <div class="col-lg-3 col-md-6 text-center wow fadeInUp" data-wow-delay="600ms">
                        <img src="./img/Tuts+.png" alt="">
                    </div>
                    <div class="col-lg-3 col-md-6 text-center wow fadeInUp" data-wow-delay="900ms">
                        <img src="./img/Microlancer copy.png" alt="">
                    </div>
                </div>
            </div>
        </div>
    </section>
    <!--end info -->
    
    <!--contact -->
    <section class="contact" id="contact">
        <div class="container">
            <div class="row text-center">
                <div class="col-lg-12">
                    <h2>Contact us</h2>
                    <p>Proin iaculis purus consequat sem cure. </p>
                    <div class="row justify-content-center">
                        <div class="col-lg-6 col-md-6"></div>
                        <div class="col-lg-6 col-md-6">
                            <div class="bg-map">
                                <img src="./img/Map Image.png" alt="">
                            </div>
                        </div>
                    </div>
                    <div class="row justify-content-center">
                        <div class="col-lg-6 col-md-6 wow fadeInLeft" data-wow-delay="100ms">
                            <div class="form">
                                <form action="">
                                    <div class="row justify-content-center">
                                        <input type="text" class="form-control" placeholder="your name* ">
                                        <input type="text" class="form-control" placeholder="your e-mail*">
                                        <input type="text" class="form-control" placeholder="subject*">
                                    </div>
                                </form>
                            </div>
                        </div>
                        <div class="col-lg-6 col-md-6 wow fadeInRight" data-wow-delay="900ms">
                            <div class="mes">
                                <div class="form">
                                    <form action="">
                                        <div class="row justify-content-center">
                                            <input type="text" class="form-control" placeholder="your message* ">
                                        </div>
                                    </form>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div class="row justify-content-center wow fadeInUp" data-wow-delay="1500ms">
                        <button class="btn text-center">
                            <p>Tell me more </p>
                        </button>
                    </div>
                </div>

            </div>
        </div>
        </div>
    </section>
    <!--end contact -->

    <!--footer -->
    <section class="footer">
        <div class="container">
            <div class="col-lg-12">
                <div class="row justify-content-center">
                    <div class="col-lg-5 col-md-6 col-sm-12">
                        <p>© Copyright 2014 FreebiesXpress.com</p>
                    </div>
                    <div class="col-lg-7 col-md-6 col-sm-12">
                        <div class="icon">
                            <i class="fa fa-twitter" aria-hidden="true"></i>
                            <i class="fa fa-facebook" aria-hidden="true"></i>
                            <i class="fa fa-instagram" aria-hidden="true"></i>
                            <i class="fa fa-google-plus" aria-hidden="true"></i>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <!--end footer -->

    <!--//scroll -->
    <a id="scrollUp" href="#top" style="position: fixed; z-index: 2147483647;"></a>

    <!--//scripts -->

    <script src="libs/js/jquery.min.js"></script>
    <script src="layout/scripts/jquery.mobilemenu.js"></script>
    <script src="libs/js/popper.js"></script>
    <script src="libs/js/bootstrap.js"></script>
    <script src="js/script.js"></script>
    <script src="libs/wow/wow.min.js"></script>
    <script src="libs/js/jquery.scrollUp.min.js"></script>

</body>

</html>